documentation/asciidoc/computers/config_txt/conditional.adoc

```diff
@@ -123,6 +123,18 @@ cmdline=cmdline-recovery.txt
 
 The raw value of the `PM_RSTS` register at bootup is available via `/proc/device-tree/chosen/bootloader/rsts` and the final partition number used for booting is available via `/proc/device-tree/chosen/bootloader/partition`. These are big-endian binary values.
 
+=== The `[boot_partition=N]` filter
+The `boot_partition` filter can be used to select alternate OS files (e.g. `cmdline.txt`) to be loaded depending on which partition `config.txt` was loaded from after processing `autoboot.txt`. This is intended for use with an `A/B` boot-system with `autoboot.txt` where it is desirable to be able to have identical files installed to the boot partition for both the `A` and `B` images.
```

Comment:
Could you add a comma between "loaded" and "depending" please?
Looks fine otherwise :+1:

Reply:
done